<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leaderboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/snake.jpg</normaloff>:/image/snake.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget
{
border-image: url(:/image/table.jpg) 0 0 0 0 stretch stretch;
}
QPushButton {
    background-color: green;
    border: 2px solid gold; 
    color: gold; 
    padding: 5px 10px; 
    border-radius: 5px; 
}

QPushButton:hover {
    background-color: darkgreen; 
    color: white; 
}

QLineEdit {
    background-color: lightgreen; 
    border: 2px solid gold; 
    color: darkgreen; 
    padding: 5px; 
    border-radius: 5px; 
}
QLineEdit:focus {
    border-color: darkgoldenrod; 
}
QTableView
{
    background-color: lightgreen; 

}
QTextBrowser {
    background-color: #8DF1F8;
    color: black; 
    font-family: Arial, sans-serif; 
    font-size: 14px; 
    line-height: 1.5; 
    padding: 10px; 
}

QTextBrowser::hover {
    background-color: #E0E0E0; 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Соединиться с сервером</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>95</width>
      <height>28</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>отправить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter a message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить список работников</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
